<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UForm_AParkDocumentsView_PlanovyiViddil</class>
 <widget class="QWidget" name="UForm_AParkDocumentsView_PlanovyiViddil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Перегляд документації</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>677</width>
        <height>633</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_dateFilter">
             <property name="text">
              <string>Поточна дата</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_filtr">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_dyspetcherska">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Інформація про показники перевезення</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_dyspetcherskaImg">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../images/UAutoParkImg.qrc">:/Splash/big/bus-256x256.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="pushButton_printPlanyValovyhDohodiv">
                  <property name="text">
                   <string>План валових доходів</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/UAutoParkImg.qrc">
                    <normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_printVidomistVykonannyaPlanuVDen">
                  <property name="text">
                   <string>Відомість виконання плану в день</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/UAutoParkImg.qrc">
                    <normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_printZvedenaVidomistPoDorojnihLystahZaMisyac">
                  <property name="text">
                   <string>Зведена відомість по дорожніх листах за місяць</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/UAutoParkImg.qrc">
                    <normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_printZvedenaVidomistPoTupahMarshrutivZaMisyac">
                  <property name="text">
                   <string>Зведена відомість по типах маршрутів за місяць</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/UAutoParkImg.qrc">
                    <normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_printZvitProZdanuVyruchkuGotivkoyu">
                  <property name="text">
                   <string>Звіт про здану виручку готівкою</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/UAutoParkImg.qrc">
                    <normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_printZvitProPerevezPilgPasajyriv">
                  <property name="text">
                   <string>Звіт про перевезення пільгових пасажирів за місяць</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/UAutoParkImg.qrc">
                    <normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</normaloff>:/function_icon_set/function_icon_set/paper_content_chart_48.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_print_perelik_pracivnykiv">
                  <property name="text">
                   <string>Друк переліку працівників</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>157</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateEdit_filtr</tabstop>
  <tabstop>pushButton_printPlanyValovyhDohodiv</tabstop>
  <tabstop>pushButton_printVidomistVykonannyaPlanuVDen</tabstop>
  <tabstop>pushButton_printZvedenaVidomistPoDorojnihLystahZaMisyac</tabstop>
  <tabstop>pushButton_printZvedenaVidomistPoTupahMarshrutivZaMisyac</tabstop>
  <tabstop>pushButton_printZvitProZdanuVyruchkuGotivkoyu</tabstop>
  <tabstop>pushButton_printZvitProPerevezPilgPasajyriv</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="../images/UAutoParkImg.qrc"/>
  <include location="../images/UAutoParkImg.qrc"/>
  <include location="../images/UAutoParkImg.qrc"/>
 </resources>
 <connections/>
</ui>
